### DISCUSSION TITLE
Github Action - description is not recognized

### DISCUSSION THREAD

**Discussion by shirady** [community/community#43299] [Actions] [answered]
### Select Topic Area

Bug

### Body

Hi,
I think It is a bug:
In [GitHub doc](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#description) there is an option to add a description (it is supposed to be required!):

> description
> Required A short description of the action

but it is not recognized (I also tried to wrap it quote and double quote and it didn't help) - see the screenshot below
![Screen Shot 2023-01-04 at 14 33 01](https://user-images.githubusercontent.com/57721533/210556711-c66b7b54-a974-436a-a76f-b59ea3a27365.png)

If I'm mistaken, please share an example with a description.

**Comment by Simran-B** [comment, marked as answer]
I believe you're confusing the documentation about programming actions (https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions) with the documentation about GitHub Actions **workflows** (https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions). Actions are building blocks for workflows, and the link you posted is about the metadata for actions. Workflows don't allow you to enter descriptions, you can only have `name`s. You could use YAML comments to add information to the source files, however:
```yaml
name: check-something
# This workflow does this and that.
```
